### DISCUSSION TITLE
open-shell-menu and windows 11

### DISCUSSION THREAD

**Discussion by buraCharles** [Open-Shell/Open-Shell-Menu#1118] [Ideas]
Open-Shell does not work with taskbar windows 11 22621.xxx
it requires Windows 10 taskbar.

**Comment by blackcrack** [comment]
nope, it requires not, it need only a update.. bring a programmer wo update
OpenShell then it is more compatible to Win10.. and READ THE OTHER DISCUSSIONS !!

best

**Comment by buraCharles** [comment]
sorry but it is inoperative with windows 11 22621.382. 
what update are you talking about?

**Comment by YesTrustMe** [comment]
another reason to avoid openshell. Sigh.

**Comment by XenHat** [comment]
Closing as duplicate conversation to meta-thead #745
